apps/meteor/client/components/MarkdownText.tsx

```diff
@@ -89,7 +88,7 @@ const MarkdownText: FC<Partial<MarkdownTextParams>> = ({
 
 	let markedOptions: marked.MarkedOptions;
 
-	const schemes = useSetting('Markdown_SupportSchemesForLink') as string;
+	const schemes = 'http,https';
```

Comment:
this value is being used to be a Regex inside the library `dompurify`, open to remove it if it's not important

Reply:
Same as the other comment about it... check the helper method `isValidUrl`